yum-requirements.txt

```diff
@@ -13,11 +13,13 @@ autoconf
 bison
 build-essential
 clang-format
+cmake
```

Comment:
I need to resurrect and finish off PR #5192. Some of these package names are invalid.